test/content/components/main_content/wolt_modal_sheet_top_bar_title_test.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:wolt_modal_sheet/src/content/components/main_content/wolt_modal_sheet_top_bar_title.dart';
+import 'package:wolt_modal_sheet/wolt_modal_sheet.dart';
+
+void main() {
+  group('WoltModalSheetTopBarTitle tests', () {
+    // Global key to be used for pageTitle widget
+    final GlobalKey pageTitleKey = GlobalKey();
+
+    testWidgets('should display top bar title when provided', (tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            body: WoltModalSheetTopBarTitle(
+              page: WoltModalSheetPage.withSingleChild(
+                child: const SizedBox.shrink(),
+                topBarTitle: const Text('Top Bar Title'),
```

Comment:
nit move these to constants outside so they can be referenced and easily changed in multiple places

Reply:
@arpadhani this is a test file. I don't think there is a reason to move these to outside.